docs/other-software/casino.md

```diff
@@ -0,0 +1,144 @@
+# CASINO
+
+!!! note
+    CASINO is not available as central install/module on ARCHER2 at this time. This 
+    page provides tips on using CASINO on ARCHER2 for users who have obtained their 
+    own copy of the code.
+
+[CASINO](https://vallico.net/casinoqmc/) is a computer program system for performing
+quantum Monte Carlo (QMC) electronic structure calculations that has been developed
+by a group of researchers initially working in the Theory of Condensed Matter group in
+the Cambridge University physics department, and their collaborators, over more than
+20 years.  It is capable of calculating incredibly accurate solutions to the Schrödinger
+equation of quantum mechanics for realistic systems built from atoms. 
+
+## Useful Links
+
+   - [CASINO User Manual](https://casinoqmc.net/casino_manual_dir/casino_manual.pdf)
+   - [CASINO Licensing](https://vallico.net/casinoqmc/request-a-copy/)
+
+## Compiling CASINO on ARCHER2
+
+You should use the `linuxpc-gcc-slurm-parallel.archer2` configuration that is supplied
+along with the CASINO source code to build on ARCHER2 and ensure that you build the
+"Shm" (System-V shared memory) version of the code.
+
+!!! bug
+    The `linuxpc-cray-slurm-parallel.archer2` configuration produces a binary that
+    crashes with a segfault and should not be used.
+
+## Using CASINO on ARCHER2
+
+The performance of CASINO on ARCHER2 is critically dependent on three things:
+
+- The MPI transport layer used: UCX is required for good scaling to multiple nodes
+- The number of cores that share System-V shared memory segments: 8 or 16 cores are
+  the best performing choices. If memory efficiency is critical, then 32 cores gives
+  good performance. More than 32 cores sharing a memory segment gives poor performance.
+- Ensuring that MPI processes are pinned to cores in a sequential manner so cores that 
+  are sharing shared memory segments are located as close to each other as possible.
+
+Next, we show how to make sure that the MPI transport layer is set to UCX, 
+how to set the number of cores sharing the System-V shared memory segments and
+how to pin MPI processes sequentially to cores.
+
+Finally, we provide an example CASINO job submission script that demonstrates all
```

Comment:
I find "an example CASINO job submission script" quite long to read. For instance, I'd use "a job submission script".

Reply:
Changed